Src/Test.UnitTests.Sarif.PatternMatcher.Cli/AnalyzeCommandTests.cs

```diff
@@ -98,7 +98,7 @@ private SarifLog RunAnalyzeCommand(string definitionsText, string fileContents)
             string searchDefinitionsPath = @$"c:\{Guid.NewGuid()}.json";
 
             var mockFileSystem = new Mock<IFileSystem>();
-
+            mockFileSystem.Setup(x => x.DirectoryExists(rootDirectory)).Returns(true);
```

Comment:
>DirectoryExists [](http://example.com/codeflow?start=40&length=15)

we have a new verification checking if the directory existing in the sarif.sdk. We will need to mock to make it to work.

Reply:
Nice, this is from Mary's change, got it.